modules/ROOT/pages/apikit-4-display-fault-task.adoc

```diff
@@ -1,28 +1,34 @@
-= To Display a Typed Fault
+= Display a Typed Fault
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-In this procedure, you make the ListInventory operation display the typed fault. For the tshirt2.wsdl file, there is just one typed fault named TshirtFault, which is mapped to every operation that the WSDL has: OrderTshirt, ListInventory, and TrackOrder.
+Make the `ListInventory` operation display the typed fault. In the tshirt2.wsdl file, there is one typed fault named *TshirtFault*, which is mapped to the operations that the WSDL has:
+
+* `OrderTshirt`
+* `ListInventory`
+* `TrackOrder`
 
 APIkit for SOAP supports the following types of SOAP faults:
 
 * Soap Fault 1.1
 * Soap Fault 1.2
 
-You can select either type using any WSDL file, even for those files that do not contain typed faults.
+You can select either type by using any WSDL file, even for files that do not contain typed faults.
 
-. In Studio, search for fault in the palette.
+. In Anypoint Studio, search for *fault* in the command palette.
 +
-Multiple SOAP Fault components might appear.
+Multiple `SOAP Fault` components can appear.
 +
-. Drag and drop the SOAP Fault component into the ListInventory:api-config flow to the right of Transform.
+. Drag and drop the `SOAP Fault` component into the `ListInventory:api-config` flow.
 +
-If the Operation Unavailable message appears telling you about a version discrepancy, cancel the operation, and choose another SOAP Fault component. Repeat these steps until you find a version match.
-. In the properties editor, in Module Configuration, select soapkit-config from the drop-down.
-. In Operation, select ListInventory from the drop-down.
-. In Fault, select `Fault` from the drop-down and save the properties
-. Click the Transform Message component to the left of the SOAP Fault component.
+If *Operation Unavailable* message appears, cancel the operation and select another `SOAP Fault` component.
+. Repeat these steps until you find a version match.
```

Comment:
This seems like a terrible UX! :)

Reply:
I agree! Requesting the customer to select as many SOAP Fault components are available until one works does not seem to be the great journey we are looking for the user.